src/Cacher.ts

```diff
@@ -1,8 +1,30 @@
+export enum CacheStatus {
+    // The book has not been cached.
```

Comment:
I might prefer JSDoc comments for these sort of things (`/** … */`) so they show up in generated documentation and in people's IDEs. I love this change otherwise though!